styleguide/source/_patterns/02-organisms/05-ec/01-ec-search-bar/_ec-search-bar.scss

```diff
@@ -1,42 +1,43 @@
-.ec-search-bar {
+.ec_search-bar {
```

Comment:
I disagree with this renaming. I believe the block's complete name should be `ec-search-bar` as it does not make sense to treat these as separable into components.